README.md

```diff
@@ -4,6 +4,8 @@
 
 ## Why?
 
+## REMEBER -- File in /php is rooted -- resset to locl
```

Comment:
Thanks!

Could we add a command to these docs that changes ownership? Like using `chown -R <GID>:<UID>`?

Also there seems to be some typos in your docs.